.rubocop.yml

```diff
@@ -8,9 +8,9 @@ AllCops:
   DisplayCopNames: true
 
 Layout/LineLength:
-  Max: 120
+  Max: 140
 Metrics/MethodLength:
-  Max: 20
+  Max: 200
```

Comment:
As mentioned in the [linters documentation](https://github.com/microverseinc/linters-config/tree/master/ruby#set-up-rubocop-github-action), kindly avoid modifying the linters configuration files because they represent style guidelines that you share with your team.